org-code-javabuilder/neighborhood/src/main/java/org/code/neighborhood/Grid.java

```diff
@@ -40,20 +40,6 @@ public GridSquare getSquare(int x, int y) {
     }
   }
 
-  // Hides all buckets from the screen
```

Comment:
did we confirm these functions should be moved? Per the student-facing api they are on Grid, which is the logical place for them to go.

Reply:
Turns out they were supposed to stay on Grid. Moved them back.